javascript/index.js

```diff
@@ -23,18 +23,152 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
+  }, (error) => console.log(error)); */
 
+// Iteration 1 - using callbacks
+getInstruction("mashedPotatoes", 0, (step1) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+  (error) => console.log(error);
+  getInstruction("mashedPotatoes", 1, (step2) => {
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
+    (error) => console.log(error);
+    getInstruction("mashedPotatoes", 2, (step3) => {
+      document.querySelector(
+        "#mashedPotatoes"
+      ).innerHTML += `<li>${step3}</li>`;
+      (error) => console.log(error);
+      getInstruction("mashedPotatoes", 3, (step4) => {
+        document.querySelector(
+          "#mashedPotatoes"
+        ).innerHTML += `<li>${step4}</li>`;
+        (error) => console.log(error);
+        getInstruction(
+          "mashedPotatoes",
+          4,
+          (step5) => {
+            document.querySelector(
+              "#mashedPotatoes"
+            ).innerHTML += `<li>${step5}</li>`;
+            (error) => console.log(error);
+            document.querySelector(
+              "#mashedPotatoes"
+            ).innerHTML += `<li>Mashed potatoes are ready</li>`;
+            document
+              .querySelector("#mashedPotatoesImg")
+              .removeAttribute("hidden");
+          },
+          (error) => console.log(error)
+        );
+      });
+    });
+  });
+});
```

Comment:
Here you have the correct logic for the callbacks but you are missing the errors placement, for example:
´´´
getInstructions("mashedpotatoes", 0, (step1) => {
   document.querySelector("#mashedPotatoes").innerHTML += '<li></li>'
       getInstructions("mashedpotatoes", 1, (step2) => {
            document.querySelector("#mashedPotatoes").innerHTML += '<li></li>'
}, (error) => console.log(error))
}, (error) => console.log(error))
```
And so on until you have all the callbacks called